wiki/Article_styling_criteria/Writing/th.md

```diff
@@ -0,0 +1,336 @@
+# การเขียน
+
+*สำหรับมาตรฐาน การจัดรูปแบบ, ดูที่: [เกณฑ์รูปแบบบทความ/การจัดรูปแบบ](../Formatting)*
+
+*ข้อสังเกต: บทความนี้ใช้ [RFC 2119](https://tools.ietf.org/html/rfc2119) เพื่ออธิบายระดับความต้องการ*
+
+ทุกบทความที่เป็นภาษาอังกฤษต้องใช้ภาษาอังกฤษแบบปกติเท่านั้น
+
+## ทะเบียนภาษา
+
+วิกิมีการเขียนสองภาษา: เป็นกลางและไม่เป็นทางการ
+
+การเขียนเป็นกลางเป็นค่าเริ่มต้นบนวิกิ
+
+การเขียนอย่างไม่เป็นทางการนั้นพิเศษและมอบให้กับบทความจำนวนเล็กน้อย ตัวอย่างบางส่วนของการเขียนนี้ ได้แก่ [คำถามที่พบบ่อย](/wiki/FAQ) และ [ศูนย์ช่วยเหลือ](/wiki/Help_Centre)
+
+สำหรับการเขียนทั้งสอง กฎทั้งหมดในส่วนนี้ของเกณฑ์ลักษณะบทความต้องปฏิบัติตาม จะมีการบันทึกข้อยกเว้นสำหรับการเขียนอย่างใดอย่างหนึ่ง
+
+## ภาษาอังกฤษ
```

Comment:
this looks irrelevant to Thai -- you are translating the article other Thai people will (ideally) use as a go-to guide

Reply:
ok

Reply:
oh yeah in thai there's no conflicting and same work stuff thing so should i put it out?

Reply:
yes